### PR TITLE
fmui https: 393 add 2 salt state to en/disable https

### PR DESCRIPTION
**Pull request by daniwe4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -318,7 +318,7 @@ This is done by certbot.
 In order for this state to run successfully, it is important that the proxy is already accessible from the Internet and the DNS entries are set.
```
Comment (tbongers-cat) [inline README.md:318]: In line 316, it should say '**proxy**-enable-https', right?

File: setup/stack/states/proxy-enable-https/https/init.sls [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+cert_renew_by_cron:
+  cron.present:
+    - user: root
+    - name: certbot renew
+    - hour: 3
+{% else %}
+custom_raise:
+  test.fail_without_changes:
+    - msg: "Missing email! Please use this command: salt 'doil.proxy' state.highstate saltenv=enable-https pillar='{\"email\": \"<your_email>\"}'"
```
Comment (tbongers-cat) [inline setup/stack/states/proxy-enable-https/https/init.sls:39]: saltenv=**proxy**-enable-https

**Review by tbongers-cat — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (daniwe4) [reply README.md:318]: Done!

File: setup/stack/states/proxy-enable-https/https/init.sls (hunk 2 above)
Comment (daniwe4) [reply setup/stack/states/proxy-enable-https/https/init.sls]: Done!

**Review by tbongers-cat — APPROVED** [review]

**Pull request merged**